content/en/blog/_posts/2021-11-02-csi-migration-status.md

```diff
@@ -0,0 +1,122 @@
+
+- storage_operation_duration_seconds
+  - This metric is exposed from the Kube-Controller-Manager. A new field `migrated` is added to the metric to indicate whether this storage operation is a CSI migration operation.
+- csi_sidecar_operations_seconds
+  - This metric is exposed from the CSI sidecars(csi-provisioner, csi-attacher, csi-resizer). A new field `migrated` is added to the metric to indicate whether this storage operation is a CSI migration operation.
+- csi_kubelet_operations_seconds
+  - This metric is exposed from Kubelet. It exposes CSI operation, driver name, status and `migrated` field to indicate whether this storage operation is a CSI migration operation.
+
+### Bug fixes and feature improvement
+
+We have been able to identify bugs like dangling attachment, garbage collection, incorrect topology label ... and we have fixed all of them to make the feature more healthy.
+
+### Cloud Provider && Cluster Lifecycle Collaboration
+
+We, SIG Storage, have been working closely with SIG CloudProvider and SIG ClusterLifeCycle to provide a better experience for the feature launching.
+
+If you are a user of public cloud provider managed Kubernetes service like GKE, EKS, AKS, there is nothing required to be done from your end. The cloud providers will manage the feature gate and ensure the feature lands correctly.
+
+If you are a user of some other kubernetes distribution services, check the official documentations to get support for the feature. For the CSI Migration feature graduation to GA, we are coordinating with SIG ClusterLifeCycle towards requesting an early planning & they have been working hard to ensure a smooth experience.
+
+## What is the timeline/status?
+
+The current and targeted releases for each individual driver is shown in the table below:
+
+| Driver           | Alpha | Beta (in-tree deprecated) | Beta (on-by-default) | GA            | Target "in-tree plugin" removal |
+| ---------------- | ----- | ------------------------- | -------------------- | ------------- | ------------------------------- |
+| AWS EBS          | 1.14  | 1.17                      | 1.23                 | 1.24 (Target) | 1.26 (Target)                   |
+| GCE PD           | 1.14  | 1.17                      | 1.23                 | 1.24 (Target) | 1.26 (Target)                   |
+| OpenStack Cinder | 1.14  | 1.18                      | 1.21                 | 1.24 (Target) | 1.26 (Target)                   |
+| Azure Disk       | 1.15  | 1.19                      | 1.23                 | 1.24 (Target) | 1.26 (Target)                   |
+| Azure File       | 1.15  | 1.21                      | 1.24 (Target)        | 1.25 (Target) | 1.27 (Target)                   |
+| vSphereVolume    | 1.18  | 1.19                      | 1.24 (Target)        | 1.25 (Target) | 1.27 (Target)                   |
```

Comment:
Given the other drivers' names are written out as phrases, I'd change `vSphereVolume` to `vSphere`.